modules/serverless-sinkbinding-yaml.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// serverless/knstive_eventing/serverless-sinkbinding.adoc
+
+[id="serverless-sinkbinding-yaml_{context}"]
+= Using SinkBinding with the YAML method
+
+ This guide describes the steps required to create, manage and delete a SInkBinding instance using YAML files.
+
+.Prerequisites
+
+* You will need to have a Knative Serving and Eventing installation.
+* You will need to have the `kn` CLI installed.
+
+.Procedure
+
+. To check that SinkBinding is set up correctly, create a Knative event display service, or event sink, that dumps incoming messages to its log.
++
+Copy the following sample YAML into a file named `service.yaml`:
++
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/knative-releases/github.com/knative/eventing-contrib/cmd/event_display
+----
++
+After you have created the `service.yaml` file, apply it by entering the following command:
++
+----
+$ oc apply --filename service.yaml
+----
+
+. Create a SinkBinding that will direct events to the service.
++
+You can do this by creating a file named `sinkbinding.yaml` and copying the following sample code into it:
++
+----
+apiVersion: sources.knative.dev/v1alpha1
+kind: SinkBinding
+metadata:
+  name: bind-heartbeat
+spec:
+  subject:
+    apiVersion: batch/v1
+    kind: Job <1>
+    selector:
+      matchLabels:
+        app: heartbeat-cron
+
+  sink:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+  ceOverrides:
```

Comment:
@matzew Why is this required ?

Reply:
I don't think it is strictly required.  It allows for outbound filtering -- which I don't think is used in this example, but it does show that you can add the overrides here and that they still exist in the `event-display`'s logs.

Reply:
Removed it from here and the `kn` command, lmk if this changes :)